protocols/gossipsub/src/types.rs

```diff
@@ -23,7 +23,7 @@ use crate::rpc_proto;
 use crate::TopicHash;
 use libp2p_core::{connection::ConnectionId, PeerId};
 use prometheus_client::encoding::EncodeLabelValue;
-use prost::Message;
+use prost::Message as ProtobufMessage;
```

Comment:
If you just need to the trait to be in scope to call functions on it you can do this:

```suggestion
use prost::Message as _;
```

Reply:
Fixed all occurrences in `gossipsub`